apps/module-smartcards/writeClerkCard.py

```diff
@@ -11,7 +11,7 @@
 f.close()
 
 election_json_bytes = json.dumps(election).encode('utf-8')
-short_value = json.dumps({'t':'clerk', 'h': hashlib.sha256(election_json_bytes).hexdigest()})
+short_value = json.dumps({'t':'admin', 'h': hashlib.sha256(election_json_bytes).hexdigest()})
```

Comment:
change the filename too? --> `writeAdminCard.py`

Reply:
Done.